c/CHANGELOG.rst

```diff
@@ -4,8 +4,13 @@
 
 **Features**
 
+- Add ``parents`` to the individual table to enable recording of pedigrees
+  (:user:`ivan-krukov`, :issue:`852`, :pr:`1125`, :pr:`866`, :pr:`1153`).
```

Comment:
I think we need :user:`benjeffery` in there too!

Reply:
Done